src/main/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -669,6 +669,8 @@ public String getHelpDoc() {
         }
         programsToRun = new LinkedHashSet<>(PROGRAM);
 
+        checkRInstallation(true);
```

Comment:
Why is this true?

Reply:
Almost all of the metrics within CollectMultipleMetrics are given a CHART_OUTPUT/Histogram_FILE which means the rscript will be run. CollectMultipleMetrics itself doesn't have those as an argument. Since those metrics will run the rscript anyways, I just put true.